features.yaml

```diff
@@ -38,8 +38,8 @@ features:
     id: :"traffic.gateway_injection"
     level:
       checklist: features/gateway_injection.md
-      maturity: Alpha
-      nextExpectedPromotion: "1.11"
+      maturity: Beta
+      nextExpectedPromotion: "1.13"
```

Comment:
Since this is going to be Beta in 1.12, based on an approval now, do we want to change the next expected promotion to 1.14? Otherwise, LGTM.